dataland-frontend/tests/component/components/resources/dataTable/MultiLayerDataTableWithEuTaxonomyNonFinancials.cy.ts

```diff
@@ -91,28 +91,26 @@ describe("Component test for the Eu-Taxonomy-Non-Financials view page", () => {
         2,
       );
 
-      getSectionHead("Revenue").click();
+      getSectionHead("Revenue");
       getCellValueContainer("Total Amount", 0)
         .invoke("text")
         .should("match", /^0\s*EUR\s.*/);
       getSectionHead("Revenue").click();
 
-      getSectionHead("CapEx").click();
-      getSectionHead("Aligned Share").click();
       getCellValueContainer("Relative Share in Percent", 1)
         .invoke("text")
-        .should("equal", `${betaTotalAlignedCapexPercentage} %`);
+        .should("contains", `${betaTotalAlignedCapexPercentage} %`);
+
       getCellValueContainer("Absolute Share", 2)
         .invoke("text")
-        .should("equal", `${gammaTotalAlignedCapexAbsoluteShareString}`);
+        .should("contains", `${gammaTotalAlignedCapexAbsoluteShareString}`);
       getCellValueContainer("Substantial Contribution to Climate Change Mitigation In Percent", 2)
         .invoke("text")
-        .should("equal", `${gammaContributionToClimateChangeMitigation} %`);
-
+        .should("contains", `${gammaContributionToClimateChangeMitigation} %`);
       getCellValueContainer("Total Amount", 2)
         .invoke("text")
         .should("match", new RegExp(`^${gammaCapexTotalAmountFormattedString}\\s.*$`));
-      getCellValueContainer("Total Amount", 2).click();
+      getCellValueContainer("Total Amount", 2).first().click();
```

Comment:
reminder to me: check situation.
generally "first()" is not a good option, since this can change very easiely and results in unwanted actions

Reply:
same as below: only click() leads to an error as cypress said there were more than one element to click. in main it was not the case. i have no strong reasoning about why this could change due to decollapsing.